client-react/src/common/UserLogo.js

```diff
@@ -1,38 +1,42 @@
 import React from "react";
 import "./UserLogo.css";
+import styled from "styled-components";
 
-export const UserLogo = props => {
+const Logo = styled.svg`
+  enablebackground: "new 0 0 24 24";
+`;
```

Comment:
do we need this? when i looked at the css in chrome, enablebackground is invalid

Reply:
that was added by ur friend not me so idk

Reply:
i removed the code and it doesn't seem affect functionality